src/serialize.js

```diff
@@ -94,7 +94,7 @@ jQuery.fn.extend({
 			var type = this.type;
 
 			// Use .is( ":disabled" ) so that fieldset[disabled] works
-			return this.name && !jQuery( this ).is( ":disabled" ) &&
+			return this.name && !jQuery.find.matchesSelector( this, ":disabled" ) &&
```

Comment:
Thanks for the feedback. JSPerf appears to be down right now, I'll put together a test when it's back.